### ISSUE TITLE
tfjs-node-gpu one tensor not getting disposed after executeAsync

### ISSUE THREAD

**Issue by iacovlev-pavel** [tensorflow/tfjs#1120]
#### TensorFlow.js version
tensorflow/tfjs-node-gpu: 0.1.21

#### Browser version
electron: 3.1.0

#### Describe the problem or feature request
I am running object detection on a set of images, 10000 in this example.
Everything get's really really slow and eventually crashes after ~4000 predictions.

After each prediction in the end I call `console.log(tf.memory());` and it seems that one of the Tensors is not disposed.
![leak](https://user-images.githubusercontent.com/1787775/51612790-d7bbb000-1f2a-11e9-89ed-73b833d292e5.png)

I can see the orphan tensors in `tf.ENV.engine.tensorInfo` but since it's a WeakMap not sure what would be the best way to handle it.
![leak2](https://user-images.githubusercontent.com/1787775/51613195-a68faf80-1f2b-11e9-856e-081ac56593d9.png)
Disposing the model does not dispose of this tensors.

Here is my weights_manifest.json: https://github.com/tensorflow/tfjs/files/2722928/model.zip

#### Code to reproduce the bug / link to feature request
```
const model = await tf.loadFrozenModel(modelPath, weightsPath);
// Runs in a loop
const shape = [1, 2560, 1920, 3];
const imageTensor = tf.fill(shape, 0, 'int32');
const predictions = await model.executeAsync(
  { image_tensor: imageTensor },
  ['detection_boxes', 'detection_scores', 'detection_classes', 'num_detections'],
);
imageTensor.dispose();
predictions.forEach(prediction => prediction.dispose());
console.log(tf.memory());
```

**Comment by iacovlev-pavel** [comment]
The leak is in the `faster_rcnn_resnet101_coco_2018_01_28` model, if I use `ssd_mobilenet_v2_coco_2018_03_29` the number of tensors is constant after each prediction.

http://download.tensorflow.org/models/object_detection/faster_rcnn_resnet101_coco_2018_01_28.tar.gz
then I used this command to convert to a web_model:
`tensorflowjs_converter --input_format=tf_saved_model --output_node_names="detection_boxes,detection_scores,num_detections,detection_classes" --saved_model_tags=serve model/faster_rcnn_resnet101_coco_2018_01_28/saved_model model/faster_rcnn_resnet101_coco_2018_01_28/web_model`

Is there any way to find out what weight causes the orphan tensor?
All I know is:
"shape": [2],
"dtype": "int32"
and that each prediction I leak 8 bytes.

There are plenty of weights in faster_rcnn that are not in ssd_mobilenet like `CropAndResize/crop_size` that match the signature.

Any way to find out what weight leaks the tensor?

**Comment by iacovlev-pavel** [comment]
rfcn_resnet101_coco_2018_01_28 leaks ever more, 18 tensors per prediction are not garbage collected.
![03](https://user-images.githubusercontent.com/1787775/51862106-8bb3a580-2346-11e9-9b5e-724d13fca76b.png)

I have tried using `tfjs-node` instead of `tfjs-node-gpu`, same amount of tensors leaked.

**Comment by iacovlev-pavel** [comment]
Looks like this is a duplicate of #1186. Fixed in 0.3.0.
